client/src/main/java/org/apache/rocketmq/client/impl/producer/DefaultMQProducerImpl.java

```diff
@@ -1124,7 +1149,12 @@ public TransactionSendResult sendMessageInTransaction(final Message msg,
                     if (null != transactionId && !"".equals(transactionId)) {
                         msg.setTransactionId(transactionId);
                     }
-                    localTransactionState = tranExecuter.executeLocalTransaction(msg, arg);
+                    if (null != localTransactionExecuter) {
```

Comment:
Why not  "localTransactionExecuter  != null"

Reply:
For keep consistency with previous code